gno.land/cmd/gnoland/testdata/maketx_call_sponsor.txtar

```diff
@@ -0,0 +1,26 @@
+# test for maketx call with sponsorship
+loadpkg gno.land/r/foo/bar $WORK
+
+# start a new node
+gnoland start
+
+## execute GetCaller
+gnokey maketx call -pkgpath gno.land/r/foo/bar -func GetCaller -gas-fee 1000000ugnot -gas-wanted 2000000 --sponsor=${USER_ADDR_test1} -broadcast -chainid=tendermint_test test1
+
+## compare GetCaller
+stdout OK!
```

Comment:
you should check the returned value.

Reply:
https://github.com/gnolang/gno/pull/2630/commits/dec14a8425ea38e40653849004d4ca969fb14194